group_vars/mons.yml.sample

```diff
@@ -43,9 +37,21 @@ dummy:
 #calamari_debug: false
 
 
-#############
-# crush rules
-#############
+##########
+# CEPHFS #
+##########
+#cephfs: cephfs # name of the ceph filesystem
+#cephfs_data: cephfs_data # name of the data pool for a given filesystem
+#cephfs_metadata: cephfs_metadata # name of the metadata pool for a given filesystem
+
+#ceph_pools:
+#  - { name: "{{ cephfs_data }}", pgs: "" }
+#  - { name: "{{ cephfs_metadata }}", pgs: "" }
```

Comment:
The placement of `ceph_pools` under CEPHFS seemed odd. Do YAML parsers concatenate multiple definitions of a list? (Not really a big deal but I could see ceph_pools being moved later on to somewhere more generic...)

Reply:
It's just because I should have called this `cephfs_pools` :). Fixing this.